contracts/fast/FastTokenFacet.sol

```diff
@@ -503,17 +574,23 @@ contract FastTokenFacet is AFastFacet, IERC20, IERC1404 {
     LibFastToken.Data storage s = LibFastToken.data();
 
     // Remove all given allowances.
-    address[] storage gaData = s.allowancesByOwner[member].values;
-    while (gaData.length > 0) {
-      // Make sure the call is performed externally so that we can mock.
-      this.performDisapproval(member, gaData[0]);
+    {
```

Comment:
Why a new scope here?

Reply:
When locals fall out of scope, they don't count on the stack anymore.
Another nice thing is that you could reuse the same variable name in a different scope - for all the compiler knows, it would be a completely different variable.

It's good practice to have variables fall out of scope as early as possible. At least it used to be the case.

Reply:
Ah nice, I had a quick Google before asking and the stack issue came up. Interesting though, thanks!